src/supporting-data/citations/components/__tests__/LiteratureCitation.spec.tsx

```diff
@@ -19,16 +23,25 @@ describe('Publication component', () => {
   });
 
   test('should expand authors', async () => {
-    expect(screen.queryByText('Ohara O.')).toBeNull();
-    fireEvent.click(screen.getByText('[...]'));
-    const author = await screen.findByText('Ohara O.');
-    expect(author).toBeTruthy();
+    expect(screen.queryByText('Ohara O.')).not.toBeInTheDocument();
+    fireEvent.click(screen.getByRole('button', { name: 'Show more' }));
+    expect(screen.getByText('Ohara O.')).toBeInTheDocument();
+  });
+
+  test('should display 1 author', async () => {
+    const mockData: CitationsAPIModel = {
+      citation: {
+        id: '14702039',
+        authors: ['Smith X.'],
+      },
+    };
+    rendered = customRender(<LiteratureCitation data={mockData} />);
```

Comment:
```suggestion
    rendered.rerender(<LiteratureCitation data={mockData} />);
```
That's the comment I did a while back that if you just keep rendering during one test, the previous render (the one in the beforeEach) will be kept, alongside this one. So you either need to unmount it, or rerender on top of the previous render, or not have a beforeEach render

Reply:
Was not causing an issue here, but could avoid unexpected bugs later on